src/lib/web3-utils.js

```diff
@@ -1,3 +1,12 @@
+export function getNetworkType(chainId) {
+  chainId = String(chainId)
+
+  if (chainId === '1') return 'main'
+  if (chainId === '4') return 'rinkeby'
```

Comment:
Is it possible to do something for staging? (cc @bpierre @sohkai)

Reply:
We could [pass a complete network](https://github.com/aragon/connect/blob/master/packages/connect-types/src/index.ts#L3-L7) to Connect in that case, rather than only the `chainId`, so we can change the `ensAddress`.

I’m also wondering if we should get rid of the name “networkType” completely, even for internal purposes? Instead we could only use `getNetworkConfig(chainId, variant)` where variant could be `undefined` or `"staging"`.

Reply:
> I’m also wondering if we should get rid of the name “networkType” completely, even for internal purposes?.

I would like to but certain ui utils seem to still need it https://github.com/aragon/aragon-ui/blob/6d1b0f40f4029585b76ddfd7ac197c03c4fa303d/src/utils/web3.js#L13-L19

Reply:
@facuspagnuolo  I added `rinkey-staging` and moved things around based on discussion with @bpierre, let me know what you think.

Reply:
@facuspagnuolo Just to double check, do you think we'll have multiple rinkeby orgs? E.g.

- Rinkeby <> Rinkeby court
- Staging (Rinkeby) <> Staging (Rinkeby) court

We would use the staging one primarily for development and leave the main Rinkeby org as a mirror of mainnet?

Reply:
If possible I'd do that, we currently have different court instances so I think it would be interesting to support the same here. If not, let's start with staging for development now and then we can swap it to the Rinkeby one that mimics Mainnet